src/styles/stars.scss

```
@@ -0,0 +1,33 @@
+.stars {
+  display: flex;
+
+  &__star {
+    display: flex;
+    flex-grow: 0;
+    flex-shrink: 1;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: auto;
+    background-image: url(../images/star.svg);
+  }
+
+  @for $sinder from 0 through 5 {
+    &--#{$sinder} > :nth-child(-n+#{$sinder}) {
+      margin-right: 4px;
+      background-image: url(../images/star-active.svg);
+    }
+  }
+
+  &--0,
+  &--1,
+  &--2,
+  &--3,
+  &--5 {
+    clip-path: circle(0);
+    height: 0;
+    padding: 0;
+  }
+}
```

```suggestion
  // Insert code that works for any count modifier
```

It's a "dirty" hack :) But it's obligatory to have all stars modifiers to work as expected!
I see that you had problems with integrating `stars` block into your `card`. But the reason of this is because you've inserted all your stars (from 0 to 5) into your card. You need to cleanup your card html code - after that rework stars count modifiers so any of them works properly.

---

виправила